kieker-analysis/src/kieker/analysisteetime/plugin/filter/forward/StringBufferFilter.java

```diff
@@ -28,11 +27,13 @@
  *
  * @author Jan Waller, Lars Bluemke
  *
- * TODO this filter does not work anymore without the array API, also it is unclear if it is useful, as all
- * strings in incoming records originate from a string registry. Therefore, they are already unique.
+ *         TODO this filter does not work anymore without the array API, also it is unclear if it is useful, as all
```

Comment:
Can we resolve this TODO before merging it?

Reply:
The filter was there in this state before I came across this filter. I personally, would delete it right away. Especially in the experimental part of analysisteetime. However, it would violate our we keep stuff for one version when it is deprecated rule.

Reply:
I am open for deleting it. I do not know why anyone has ported this filter, as it was no longer necessary since at least 1.12.

Reply:
Nothing is using it.